packages/desktop-electron/beforePackHook.ts

```diff
@@ -26,6 +27,16 @@ const beforePackHook = async (context: AfterPackContext) => {
     });
 
     console.info(`Rebuilt better-sqlite3 with ${arch}!`);
+
+    if (context.packager.platform.name === 'windows') {
+      console.info(`Windows Store build - copying appx files...`);
```

Comment:
There's no hook that can identify the appx build, so we have to do it for all windows builds 😢